scripts/background/Utility.js

```diff
@@ -2,7 +2,7 @@
 
 'use strict';
 
-var DEBUG_FLAG = false;
+var DEBUG_FLAG = true;
```

Comment:
Looks like this flag may need to be set back to `false`

Reply:
Reverted